lib/ansible/modules/cloud/misc/rhevm.py

```diff
@@ -591,7 +591,6 @@ def set_Disk(self, diskname, disksize, diskinterface, diskboot):
                 return False
         elif int(DISK.size) < (1024 * 1024 * 1024 * int(disksize)):
```

Comment:
There is another bug here. if block above is similar to elif block so I doubt this ever worked. Could you please correct the block as well (maybe comparison sign in elif ) ?